vajram/vajram-samples/src/main/java/com/flipkart/krystal/vajram/samples/benchmarks/calculator/FormulaImpl123.java

```diff
@@ -0,0 +1,106 @@
+package com.flipkart.krystal.vajram.samples.benchmarks.calculator;
+
+import static com.flipkart.krystal.data.ValueOrError.valueOrError;
+import static com.google.common.collect.ImmutableMap.toImmutableMap;
+
+public final class FormulaImpl123 {
```

Comment:
Let's delete this file?

Reply:
I have kept for reference for now. Will delete post review.